src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowserLarge.tsx

```diff
@@ -93,7 +93,7 @@ const ArtworkImageBrowserLarge: React.FC<ArtworkImageBrowserLargeProps> = ({
             my={2}
             maxHeight={maxHeight}
             artwork={artwork}
-            activeIndex={index}
+            activeIndex={artwork.setVideoAsCover ? index - 1 : index}
```

Comment:
non-blocking[nitpick]: I would alias artwork as figures for the sake of consistency.

Reply:
Not sure I follow, can you elaborate? The only thing that has changed in this file is add another field to our relay query, which I read in here

Reply:
Oh! Nevermind... ignore this. By the time we get here videos are already filtered out of the array and sent elsewhere, so these are just images. 🧠✨

Reply:
Yep yep, all the previous work in MP worked out well here. (Even changes there were super minimal.)